examples/lock-app/mbed/README.md

```diff
@@ -2,200 +2,258 @@
+> information about using VSCode in container.**
 
-# Building and Flashing Example Application
+To initialize development environment, download all registered sub-modules and
+activate development environment:
 
-## Development environment setup    
-
-Building the example application requires the use of **ARM Mbed-OS** sources 
-and **arm-none-gnu-eabi** toolchain. The OpenOCD package is used for 
-flashing and debugging purposes.
-<br>
-Some additional packages may be needed, depending on selected build target 
-and its requirements.
-
-> **The VSCode devcontainer has these components pre-installed. Using 
-the VSCode devcontainer is the recommended way to interact with ARM Mbed-OS 
-port of the Matter Project.**   
->    
-> **Please read this [README.md](../../..//docs/VSCODE_DEVELOPMENT.md) for 
-more information about using VSCode in container.**
-
-To initialize development environment, download all registered sub-modules 
-and activate development environment:
 ```
-$ source ./scripts/bootstrap.sh    
+$ source ./scripts/bootstrap.sh
 $ source ./scripts/activate.sh
 ```
-If packages are already installed then simply just activate development 
+
+If packages are already installed then simply just activate development
 environment:
+
 ```
 $ source ./scripts/activate.sh
 ```
+
 ## Building
 
-The Lock-app application can be built in the same way as any other Matter 
-example ported to mbed-os platform.
-<br>
-* **by using generic vscode task**:  
+The Lock application can be built in the same way as any other Matter example
+ported to the mbed-os platform.
+
+-   **by using generic vscode task**:
+
+```
+Command Palette (F1) => Run Task... => Run Mbed application => build => lock-app => (board name) => (build profile)`
 ```
-Command Palette (F1)  => Tasks: Run Task => Run Mbed application => build => (example name) => (board name) => (build profile)`
-```  
-<br>
 
-* **by calling explicitly building script:**
+-   **by calling explicitly building script:**
+
 ```
-$ ${MATTER_ROOT}/scripts/examples/mbed_example.sh -c build -a <example name> -b <board name> -p <build profile>
+${MATTER_ROOT}/scripts/examples/mbed_example.sh -c=build -a=lock-app -b=<board name> -p=<build profile>
 ```
-Both approaches are limited to supported evaluation boards which are 
-listed in [Supported devices](#Supported-devices) paragraph. 
-<br>When using building script, it is possible expand its list of acceptable 
-targets which may be useful for rapid testing of a new mbed target.
-<br><br>For more detailed description about porting mbed targets into the 
-Matter project, please read **[LINK HERE]**
+
+Both approaches are limited to supported evaluation boards which are listed in
+[Supported devices](#supported_devices) paragraph.
+
+Mbed OS defines three building profiles: _develop, debug_ and _release_. For
+more details please visit
+[ARM Mbed OS build profiles](https://os.mbed.com/docs/mbed-os/v6.13/program-setup/build-profiles-and-rules.html).
```

Comment:
can we use /latest/ rather than v6.13 in link? (just check it hasn't changed first)